tests/js/spec/views/dashboardsV2/widgetQueries.spec.jsx

```diff
@@ -302,6 +320,114 @@ describe('Dashboards > WidgetQueries', function () {
     expect(childProps.tableResults[1].data[0].title).toBeDefined();
   });
 
+  it('can send big number result queries', async function () {
+    const tableMock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/eventsv2/',
+      body: {
+        meta: {'sdk.name': 'string'},
+        data: [{'sdk.name': 'python'}],
+      },
+    });
+
+    let childProps = undefined;
+    const wrapper = mountWithTheme(
+      <WidgetQueries
+        api={api}
+        widget={{
+          title: 'SDK',
+          interval: '5m',
+          displayType: 'big_number',
+          queries: [{conditions: 'event.type:error', fields: ['sdk.name'], name: 'sdk'}],
+        }}
+        organization={initialData.organization}
+        selection={selection}
+      >
+        {props => {
+          childProps = props;
+          return <div data-test-id="child" />;
+        }}
+      </WidgetQueries>,
+      initialData.routerContext
+    );
+    await tick();
+    await tick();
+
+    // Child should be rendered and 2 requests should be sent.
+    expect(wrapper.find('[data-test-id="child"]')).toHaveLength(1);
+    expect(tableMock).toHaveBeenCalledTimes(1);
+    expect(tableMock).toHaveBeenCalledWith(
+      '/organizations/org-slug/eventsv2/',
+      expect.objectContaining({
+        query: expect.objectContaining({
+          referrer: 'api.dashboards.bignumberwidget',
+          query: 'event.type:error',
+          name: 'SDK',
+          field: ['sdk.name'],
+          statsPeriod: '14d',
+          environment: ['prod'],
+          project: [1],
+        }),
+      })
+    );
+    expect(childProps.timeseriesResults).toBeUndefined();
+    expect(childProps.tableResults[0].data).toHaveLength(1);
+    expect(childProps.tableResults[0].meta).toBeDefined();
+  });
+
+  it('can send world map result queries', async function () {
+    const tableMock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/events-geo/',
+      body: {
+        meta: {'sdk.name': 'string'},
+        data: [{'sdk.name': 'python'}],
+      },
+    });
+
+    let childProps = undefined;
+    const wrapper = mountWithTheme(
+      <WidgetQueries
+        api={api}
+        widget={{
+          title: 'SDK',
+          interval: '5m',
+          displayType: 'world_map',
+          queries: [{conditions: 'event.type:error', fields: ['sdk.name'], name: 'sdk'}],
```

Comment:
This test feels like it might bite us in the future since this isn't a query the world map widget would actually send right?

Reply:
I'll change it to `count()`